azkaban-common/src/main/java/azkaban/server/AzkabanServer.java

```diff
@@ -118,17 +132,14 @@ private static Props loadConfigurationFromAzkabanHome() {
       logger.error("AZKABAN_HOME not set. Will try default.");
       return null;
     }
-
-    if (!new File(azkabanHome).isDirectory()
-        || !new File(azkabanHome).canRead()) {
+    if (!new File(azkabanHome).isDirectory() || !new File(azkabanHome).canRead()) {
```

Comment:
thanks for reformatting this. maybe we will need to rely on intellij plugin to do this.